config/owasp/suppressions.xml

```diff
@@ -2,14 +2,26 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
 
   <suppress>
-    <gav regex="true">^com\.google\.guava:guava:.*$</gav>
-    <cpe>cpe:/a:google:guava</cpe>
+    <notes><![CDATA[Used only for testing not used in prod]]></notes>
+    <filePath regex="true">.*postgresql-1\.(10\.6|11\.3)\.jar</filePath>
+    <cpe regex="true">cpe:/a:postgresql:postgresql:1\.(10\.6|11\.3)</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[Used only for testing not used in prod]]></notes>
+    <filePath regex="true">^org\.scala-lang:scala-(library|reflect|compiler):.*$</filePath>
+    <cpe>cpe:/a:scala-lang:scala</cpe>
   </suppress>
 
   <suppress>
     <notes><![CDATA[Used only for testing not used in prod]]></notes>
-    <filePath regex="true">.*postgresql-1\.10\.6\.jar</filePath>
-    <cpe>cpe:/a:postgresql:postgresql:1.10.6</cpe>
+    <gav regex="true">^org\.scala-lang:scala-(compiler|reflect|library):.*$</gav>
```

Comment:
could probably just suppress everything with scala? it's only used for perf testing

Reply:
In 2 minds about that as if it gets added later it's easy to miss that everything is suppressed. Even if it really should  not be added in the first place outside testing :-)